NAMapKit/NATiledImageMapView.m

```diff
@@ -32,7 +32,7 @@ - (id)initWithFrame:(CGRect)frame tiledImageDataSource:(NSObject <ARTiledImageVi
 - (void)createImageView
 {
     if (self.dataSource) {
-        _imageView = [[ARTiledImageView alloc] initWithDataSource:self.dataSource];
+        self.imageView = [[ARTiledImageView alloc] initWithDataSource:self.dataSource];
```

Comment:
this one line of code is why the travis checks failed over the weekend. as ive said, im a noobie dev so it could be wrong.